### PR TITLE
[NEW] Complete sidebar sections and new style 

### PR DESCRIPTION
**Pull request by tiagoevanp**
## Jira tasks: [MKP-209](https://rocketchat.atlassian.net/browse/MKP-209?atlOrigin=eyJpIjoiZjAxMjNiODVlMGY1NDliM2FhZWIxMDcxZTBiODlkMzUiLCJwIjoiaiJ9) && [MKP-214](https://rocketchat.atlassian.net/browse/MKP-214?atlOrigin=eyJpIjoiZDE3OWNjMTFjYWRiNDYyZDhkZGViNzFiNzFlNjA3ODciLCJwIjoiaiJ9)

Add all the different sections from Marketplace sidebar;
Update the style of Admin/Omnichannel/Marketplace Sidebar;
Demo image:
image

## Further comments

MKP-209
MKP-214

[MKP-209]: https://rocketchat.atlassian.net/browse/MKP-209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/meteor/client/views/marketplace/AppsProvider.tsx [hunk 1]
```diff
@@ -241,19 +271,38 @@ const AppsProvider: FC = ({ children }) => {
 				apps: installedAppsData,
 			});
 		}
+
+		if (!privateAppsError) {
+			if (privateApps.length) {
```
Comment (rique223) [inline apps/meteor/client/views/marketplace/AppsProvider.tsx:276]: Using typescripts boolean casting here may be a good idea, would make the code more readable. e.g.: if (Boolean(privateApps.length))

Comment (tiagoevanp) [reply apps/meteor/client/views/marketplace/AppsProvider.tsx:276]: hmmmm, ok... I will put a `privateApps.length > 0`, sounds better indeed

File: apps/meteor/client/views/marketplace/sidebarItems.ts [hunk 2]
```diff
@@ -8,17 +8,35 @@ export const {
 } = createSidebarItems([
 	{
 		href: 'marketplace/explore/list',
-		icon: 'cube',
+		icon: 'compass',
+		textColor: 'stroke-extra-dark',
```
Comment (tassoevan) [inline apps/meteor/client/views/marketplace/sidebarItems.ts:12]: Better replace this stylistic property with something more semantic.

File: apps/meteor/client/views/marketplace/AppsProvider.tsx [hunk 3]
```diff
@@ -351,6 +412,17 @@ const AppsProvider: FC = ({ children }) => {
...
 
+				if (app.private) {
+					dispatchPrivateApps({
+						type: 'update',
+						app: {
+							...app,
+							status,
+						},
+						reload: fetch,
+					});
```
Comment (tassoevan) [inline apps/meteor/client/views/marketplace/AppsProvider.tsx:423]: No `return`? It's supposed to dispatch twice?

File: apps/meteor/client/views/marketplace/AppsProvider.tsx [hunk 4]
```diff
@@ -322,6 +375,14 @@ const AppsProvider: FC = ({ children }) => {
 					return;
 				}
 
+				if (app.private) {
+					dispatchPrivateApps({
+						type: 'delete',
+						appId,
+						reload: fetch,
+					});
```
Comment (tassoevan) [inline apps/meteor/client/views/marketplace/AppsProvider.tsx:383]: No `return`? It's supposed to dispatch twice?

File: apps/meteor/client/views/marketplace/AppsProvider.tsx [hunk 5]
```diff
@@ -303,6 +352,10 @@ const AppsProvider: FC = ({ children }) => {
 				return;
 			}
 
+			if (privateApp) {
+				dispatchPrivateApps({ type: 'update', app: privateApp, reload: fetch });
```
Comment (tassoevan) [inline apps/meteor/client/views/marketplace/AppsProvider.tsx:356]: No `return`? It's supposed to dispatch twice?

**Review by tassoevan — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/marketplace/AppsProvider.tsx (hunk 4 above)
Comment (tiagoevanp) [reply apps/meteor/client/views/marketplace/AppsProvider.tsx:383]: two different lists

File: apps/meteor/client/views/marketplace/AppsProvider.tsx (hunk 5 above)
Comment (tiagoevanp) [reply apps/meteor/client/views/marketplace/AppsProvider.tsx:356]: two different lists

File: apps/meteor/client/views/marketplace/AppsProvider.tsx (hunk 3 above)
Comment (tiagoevanp) [reply apps/meteor/client/views/marketplace/AppsProvider.tsx:423]: two different lists

**Review by tassoevan — APPROVED** [review]

**Pull request merged**